--- php-5.4.22.a/configure	2013-11-13 09:06:41.000000000 +0200
+++ php-5.4.22.b/configure	2013-11-29 23:47:10.000000000 +0200
@@ -52548,7 +52548,7 @@
     }
 
 _ACEOF
-if ac_fn_c_try_run "$LINENO"; then :
+#if ac_fn_c_try_run "$LINENO"; then :
 
     LIBS=$old_LIBS
 
@@ -52556,16 +52556,16 @@
 $as_echo "yes" >&6; }
 
 
-else
+#else
 
-    LIBS=$old_LIBS
+#    LIBS=$old_LIBS
 
-      { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-      as_fn_error $? "build test failed. Please check the config.log for details." "$LINENO" 5
+#      { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
+#$as_echo "no" >&6; }
+#      as_fn_error $? "build test failed. Please check the config.log for details." "$LINENO" 5
 
 
-fi
+#fi
 rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
   conftest.$ac_objext conftest.beam conftest.$ac_ext
 fi
