diff -urbN php-5.3.28.a/configure php-5.3.28.b/configure
--- php-5.3.28.a/configure	2014-01-12 02:23:54.000000000 +0200
+++ php-5.3.28.b/configure	2014-01-12 02:25:00.000000000 +0200
@@ -38277,7 +38277,7 @@
   if test "$PHP_FREETYPE_DIR" != "no"; then
 
     for i in $PHP_FREETYPE_DIR /usr/local /usr; do
-      if test -f "$i/include/freetype2/freetype/freetype.h"; then
+      if test -f "$i/include/freetype2/freetype.h"; then
         FREETYPE2_DIR=$i
         FREETYPE2_INC_DIR=$i/include/freetype2
         break
